src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpRequestStream.cs

```diff
@@ -110,7 +110,7 @@ public override Task WriteAsync(byte[] buffer, int offset, int count, Cancellati
 
             if (count > buffer.Length - offset)
             {
-                throw new ArgumentException(nameof(buffer));
+                throw new ArgumentException(SR.Argument_BufferNotLongEnough, nameof(buffer));
```

Comment:
Opened via the System.Net.Http solution in `corefx/src/System.Net.Http`.

I'm assuming the pulled-into-elsewhere is affecting several other projects in top-level folders that appear to not have a solution themselves, but it can be hard to find the links sometimes, if they exist.